src/test/java/com/algorand/algosdk/integration/Stepdefs.java

```diff
@@ -660,21 +745,33 @@ public void walletInfo() throws com.algorand.algosdk.kmd.client.ApiException, No
     }
 
     @Given("default transaction with parameters {int} {string}")
-    public void defaultTxn(int amt, String note) throws ApiException, NoSuchAlgorithmException{
+    public void defaultTxn(int amt, String note) throws ApiException, NoSuchAlgorithmException {
+        defaultTxnWithAddress(amt, note, getAddress(0));
+
+        pk = getAddress(0);
```

Comment:
could `pk` initialization be rolled into the `defaultTxnWithAddress` method?

Reply:
@algochoi Yes - That's a holdover from prior iterations.  I attempted to address via https://github.com/algorand/java-algorand-sdk/pull/350/commits/e67d6a95fa1a08a2252ffba4129dd9f27c867727.